src/components/createForms/Create.jsx

```diff
@@ -70,7 +79,7 @@ const Create = (props) => {
               <Button text="Preview" onClick={onSubmit} type="solid" />
             </Col>
             <Col md={{ span: 1, offset: 1 }}>
-              <Button onClick={onDelete} text="Delete" />
+              <Button onClick={onCancel} text="Cancel" />
```

Comment:
Could you rename this to `cancel` or something similar? The event function shouldn't be named like an event. `onClick={cancel}` would look better.

Reply:
That is form @leksyib's PR on creating article. I checked it to `onCancel` because it might be confused with the actual article deleting which I implemented.